packages/paypal-express-integration/.eslintrc.json

```diff
@@ -0,0 +1,46 @@
+{
+    "extends": ["../../.eslintrc.json"],
+    "ignorePatterns": ["!**/*"],
+    "overrides": [
+        {
+            "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+            "rules": {
```

Comment:
Can we try and fix some relatively fixable issues? Or will be looking at them later?